src/lib/config.ts

```diff
@@ -115,9 +115,8 @@ class Config {
    * @param {IConfig} - config object to store as config file
    */
   private async write(config: IConfig): Promise<void> {
-    return this.fs.writeFile({
-      path: this.configFullPath,
-      contents: JSON.stringify(config, null, 2)
+    return await this.writeFile(this.configFullPath, config).catch(error => {
```

Comment:
I think it can be just `return this.writeFile(this.configFullPath, config)` or `invoke(...)` directly without `.catch` and throw

Reply:
I agree, that was also what I had in mind, and to my tests, it works like that